pom.xml

```diff
@@ -11,8 +11,8 @@
 	<url>https://projects.spring.io/spring-cloud/</url>
 
 	<modules>
-		<module>spring-boot-generate</module>
 		<module>spring-scaffold-cli</module>
+		<module>spring-boot-generate</module>
```

Comment:
Consider using a more specific module name instead of a generic one like `spring-boot-generate`. ### Example: `<module>spring-boot-my-application</module>`